lexicons/tools/ozone/set/defs.json

```diff
@@ -0,0 +1,49 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.defs",
+  "defs": {
+    "set": {
+      "type": "object",
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 3,
+          "maxLength": 128
```

Comment:
Just vibes, but this number sounds a little low.

Reply:
based on @bnewbold 's proposal so I'll wait for him to chime in.

Reply:
I'm ok with it being low. this is basically a "slug" variable name, or similar to a record-key (though I wouldn't use the record key string format specifically). probably the backend implementation should enforce that it has no whitespace when doing an upsert? and we could describe that here in a description in the lexicon.